deployment/k3s/templates/deployments/backend/calendar-cronjob.yaml

```diff
@@ -0,0 +1,42 @@
+{{ range $scrape_task := $.Values.calendar.scrape_tasks }}
```

Comment:
before merging, we have to add a feature gate for this, to only be available in master.
This is currently not done, because Otherwise this would be untestable